frontend/src/components/pages/agents/details/a2a/chat/ai-agent-chat.tsx

```diff
@@ -53,24 +51,39 @@ export const AIAgentChat = ({ agent }: AIAgentChatProps) => {
   });
 
   return (
-    <div className="flex h-full w-full flex-col">
-      <Conversation className="relative flex flex-1 flex-col">
-        <ConversationContent>
+    <div className="flex flex-col h-[calc(100vh-255px)]">
+      {/* Context ID header */}
+      {contextId && (
+        <div className="shrink-0 border-b bg-muted/30 px-4 py-2">
+          <div className="flex gap-1.5 text-muted-foreground text-xs">
+            <span className="font-medium">context_id:</span>
+            <span className="font-mono">{contextId}</span>
```

Comment:
Let's use `Typography` component we created from `frontend/src/components/redpanda-ui/components/typography.tsx`